addons/point_of_sale/static/tests/tours/TicketScreen.tour.js

```diff
@@ -234,3 +234,29 @@ registry.category("web_tour.tours").add("LotRefundTour", {
             ProductScreen.checkFirstLotNumber("123456789"),
         ].flat(),
 });
+
+registry.category("web_tour.tours").add("RefundFewQuantities", {
+    test: true,
+    url: "/pos/ui",
+    steps: () =>
+        [
+            Dialog.confirm("Open session"),
+            ProductScreen.clickShowProductsMobile(),
+            ProductScreen.clickDisplayedProduct("Sugar"),
+            ProductScreen.pressNumpad("0", "."),
+            ProductScreen.selectedOrderlineHas("Sugar", "0.00", "0.00"),
+            ProductScreen.pressNumpad("0", "2"),
+            ProductScreen.selectedOrderlineHas("Sugar", "0.02", "0.06"),
+            ProductScreen.clickPayButton(),
+            PaymentScreen.clickPaymentMethod("Bank"),
+            PaymentScreen.clickValidate(),
+            ReceiptScreen.isShown(),
+            ReceiptScreen.clickNextOrder(),
+            ProductScreen.clickRefund(),
+            TicketScreen.selectOrder("-0001"),
+            ProductScreen.pressNumpad("0", ".", "0", "2"),
```

Comment:
Looking at the rest of the code we never use `pressNumpad` with more than 2 inputs. So we might want to do the same here. Beside that this LGTM